functional/unit_action_test.go

```diff
@@ -313,6 +313,9 @@ func replaceUnitCommon(cmd string) error {
 	if len(units) != 1 {
 		return fmt.Errorf("Did not find 1 unit in cluster: \n%s", stdout)
 	}
+	if err := waitForActiveUnitsReplaceCmds(cluster, m, cmd, 1); err != nil {
+		return err
+	}
```

Comment:
@dongsupark I think there has been a misunderstanding here: my point was not to invoke the wait function after already having checked that we have the expected number of units -- that won't make it any more reliable as far as I can tell. Rather, my point was that *instead* of a one-off check directly after the command, it is probably better to use the wait function to see whether the expected state is reached *eventually*... At least that's what all the existing tests I have looked at so far seem to be doing.

(Of course that only applies to cases where we are actually expecting units to become active respectively to cease being active...)